docs/pages/getting-started.md

```diff
@@ -0,0 +1,39 @@
+# Getting started
+When you have installed all the [prerequisites](/prerequisites.md) you are good to go.
+
+Start by installing the Simple Fable template:
+```
+dotnet new --install "Fable.Template::*"
+```
+Create a directory for your project:
+```
+mkdir FableApp
+cd FableApp
+```
+Scaffold a Fable app from the template:
+```
+dotnet new Fable
+```
+This will create Fable project in this directory. Now install the npm dependencies using yarn:
```

Comment:
Add a note to advice users to always check the README file in case the instructions to install dependencies and run the project have changed for the template.